tests/PortingAssistant.Client.IntegrationTests/RunAnalysisCorrectnessWithDotNetFramework.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Diagnostics;
+using NUnit.Framework;
+using PortingAssistant.Client.IntegrationTests.TestUtils;
+
+namespace PortingAssistant.Client.IntegrationTests
+{
+    public class RunAnalysisCorrectnessWithDotNetFrameworkTests
+    {
+        private string tmpTestProjectsExtractionPath;
+        private string testSolutionPath;
+        private string expectedAnalysisResultRootDir;
+        private string actualAnalysisResultRootDir;
+
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            expectedAnalysisResultRootDir = Path.Combine(
+                TestContext.CurrentContext.TestDirectory, "TestProjects");
+
+            tmpTestProjectsExtractionPath = Path.GetFullPath(Path.Combine(
+                TestContext.CurrentContext.TestDirectory,
+                Path.GetRandomFileName()));
+            Directory.CreateDirectory(tmpTestProjectsExtractionPath);
+            string testProjectZipPath = Path.Combine(
+                TestContext.CurrentContext.TestDirectory,
+                "TestProjects",
+                "NetFrameworkExample.zip");
+            using (ZipArchive archive = ZipFile.Open(
+                testProjectZipPath, ZipArchiveMode.Read))
+            {
+                archive.ExtractToDirectory(tmpTestProjectsExtractionPath);
+            }
+
+            actualAnalysisResultRootDir = tmpTestProjectsExtractionPath;
```

Comment:
That's a very good idea, thank you for the suggestion. For now, since the goal is mainly verifying the correctness and flow of the PA client, one test solution would be sufficient to validate the overall flow. Tho I'll keep a note to use config file if we are increasing the complexity and scope of the tests in the future.